synapse/storage/data_stores/main/schema/delta/58/persist_ui_auth.sql

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2020 The Matrix.org Foundation C.I.C
```

Comment:
can you give the delta file a number please, as per https://github.com/matrix-org/synapse/blob/develop/synapse/storage/data_stores/main/schema/delta/58/02remove_dup_outbound_pokes.sql? `03persist_ui_auth.sql` or so.